MAUI/Scheduler/agenda-view.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: post
+title: Agenda view in .NET MAUI Scheduler control | Syncfusion
+description: Learn here all about the Agenda view feature of Syncfusion .NET MAUI Scheduler (SfScheduler) control and more.
+platform: maui
+control: SfScheduler
+documentation: ug
+---
+
+# Agenda view in .NET MAUI Scheduler (SfScheduler)
+
+The agenda view displays the events or appointments (normal appointments, all-day appointments, recurring appointments, and span appointments) in chronological order, grouped by date between the minimum and maximum dates and the current date displayed by default. When the `AppointmentsSource` property of `SfScheduler` is `null,` the agenda view will show only the month, week, and date headers.
```

Comment:
1. `null,` -> `null`,
2. When the `AppointmentsSource` property of `SfScheduler` is `null,` the agenda view will show only the month, week, and date headers. **[ in mobile view and responsive UI] and for web view ?**

Reply:
Updated content for the desktop view as well.